xsd/UFTP-agr-dso.xsd

```diff
@@ -59,8 +64,7 @@
     <xs:complexType name="D-PrognosisISPType">
         <xs:attribute name="Power" type="xs:integer" use="required">
             <xs:annotation>
-                <xs:documentation>Power specified for this ISP in Watts. Also see the important notes about the sign of this attribute in the main documentation entry for the ISP element.</xs:documentation>
-            </xs:annotation>
+                <xs:documentation>Power specified for this ISP in the given PowerUnit. Also see the important notes about the sign of this attribute in the main documentation entry for the ISP element.</xs:documentation>            </xs:annotation>
```

Comment:
Since the new property is optional in this Pull Request, I would suggest adding something along the lines of: (If the field is missing, it should be interpreted as Watt) after PowerUnit.